opentype-shaping-malayalam.md

```diff
@@ -68,8 +68,7 @@ A syllable's base consonant is generally rendered in its full form
 syllable frequently take on secondary forms. Different GSUB
 substitutions may apply to a script's **pre-base** and **post-base**
 consonants. Some of these substitutions create **above-base** or
-**below-base** forms. The **Reph** form of the consonant "Ra" is an
-example.
+**below-base** forms.
```

Comment:
Thanks for catching this. However, this is the "explain the terminology" section, which we're trying to keep as identical as possible between all the scripts covered by the Indic2 model. So we do have to introduce the term "Reph" somewhere in here, because it occurs later on in places like the syllable-identifying regexes, base-search, and the GSUB feature. Well, lots of places.

Also, rather than just removing the erroneous "Reph" reference here (as an above/below-base example), I would prefer to replace it with an appropriate example.

So, probably, put in a reference to the below-base La form, which would definitely be helpful. Then we just figure out some wording to note that "Reph" is going to come up because of its treatment in other scripts....

Reply:
There is a fuller explanation of Dot Reph and Chillu R down in stage 1, a bit before the regex section. That might be too wordy to duplicate here, though.

Reply:
Ok, it is better to place the reference to `Reph` along with `pre-base` and `post-base`; because in Malayalam reph appear normally as post-base (traditional orthography) or pre-base (reformed orthography). I will make an update.